packages/server/src/api/controllers/deploy/aws.js

```diff
@@ -63,8 +64,22 @@ function walkDir(dirPath, callback) {
   }
 }
 
+function prepareUploadForS3({ filePath, s3Key, metadata, s3 }) {
+  const fileExtension = [...filePath.split(".")].pop()
+  const fileBytes = fs.readFileSync(filePath)
+  return s3
+    .upload({
+      Key: s3Key,
+      Body: fileBytes,
+      ContentType: CONTENT_TYPE_MAP[fileExtension],
```

Comment:
Everywhere else you have lowercase'd the file extension, I don't know if `filePath` has already been lowercase'd here but it might be worth doing it to the file extension so the map lookup won't fail on say `PNG`

Reply:
Great catch - that would be a wee bug there!